CONTRIBUTING.md

```diff
@@ -1,36 +1,41 @@
 # Contributing
 
 The dance tool and tests run on the host; macOS, Linux, and Windows are expected to work.
-Databases under test (FerretDB and MongoDB) may be running on the host or inside Docker; Docker Compose configuration is provided for convenience, but not required.
-In particular, the FerretDB development cycle (fix-compile-run-dance) is faster with it running on the host as it does not involve Docker images building or PostgreSQL restarts.
+Databases under test (FerretDB and MongoDB) may be running on the host or inside Docker; 
+Docker Compose configuration is provided for convenience, but not required.
+In particular, the FerretDB development cycle (fix-compile-run-dance) is faster with it running on the host 
+as it does not involve Docker images building or PostgreSQL restarts.
 
 ## Running tests
 
 ```sh
-make dance DB=ferretdb TEST=ferret
+DB=ferretdb TEST=ferret bin/task dance
 ```
 
-That command will run `ferret` tests ([configuration](https://github.com/FerretDB/dance/blob/main/tests/ferret.yml), [test](https://github.com/FerretDB/dance/tree/main/tests/ferret)) against FerretDB.
+That command will run `ferret` tests ([configuration](https://github.com/FerretDB/dance/blob/main/tests/ferret.yml), 
+[test](https://github.com/FerretDB/dance/tree/main/tests/ferret)) against FerretDB.
 `DB` environment variable should have the value `ferretdb` or `mongodb`.
 It defines what tests are expected to pass and fail.
-For example, see [mongo-go-driver tests configuration](https://github.com/FerretDB/dance/blob/main/tests/mongo-go-driver.yml) (fields under `results.ferretdb` and `results.mongodb`).
+For example, see [mongo-go-driver tests configuration](https://github.com/FerretDB/dance/blob/main/tests/mongo-go-driver.yml)
+(fields under `results.ferretdb` and `results.mongodb`).
 `TEST` environment variable should have the value `ferret`, `mongo-go-driver`, or be empty.
 It defines what test configuration to run; empty value runs all configurations.
 
 ## Starting environment with Docker Compose
 
 ```sh
-make env-up DB=mongodb
+DB=mongodb bin/task env-up
```

Comment:
I set env variable before `bin/task` as it wouldn't recognize it.

Reply:
https://taskfile.dev/#/usage?id=variables says:

> Since some shells don’t support above syntax to set environment variables (Windows) tasks also accepts a similar style when not in the beginning of the command.

```sh
$ task write-file FILE=file.txt "CONTENT=Hello, World!" print "MESSAGE=All done!"
```

So let's double check that

Reply:
There was some strange error with precondition check that's why I thought it's not working. Moved it back.